### PR TITLE
User story parent create

### PR DESCRIPTION
**Pull request by krislitman**
Added a link on the welcome page for Dealers. Also added a Dealer index, show, and new page with the routes updated. Dealers controller added, and views for all pages have been created. Form has been made to create a new Dealer and will need to work on posting the action!

### CODE REVIEW

File: app/views/dealers/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<a href="/dealers">Dealer Index</a>
+
+<h1><%= @dealer.name %></h1>
+<p><%= @dealer.fully_staffed %></p>
+<p><%= @dealer.monthly_gross %></p>
+<p><%= @dealer.city %></p>
+<p><%= @dealer.created_at %></p>
+<p><%= @dealer.updated_at %></p>
```
Comment (abreaux26) [inline app/views/dealers/show.html.erb]: @duguy8 Do you think we need these here?

File: app/views/welcome/index.html.erb [hunk 2]
```diff
@@ -1,6 +1,6 @@
-<h1>Welcome to the Relational Rails</h1>
+<h1>Welcome to Relational Rails</h1>
```
Comment (abreaux26) [inline app/views/welcome/index.html.erb:1]: Nice catch!!

File: app/views/welcome/index.html.erb [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
 
 <ul>
-  <li><a href="/schools">Schools</a></li>
-  <!-- <li><a href="/tasks/new">New Task</a></li> -->
```
Comment (abreaux26) [inline app/views/welcome/index.html.erb:5]: Thanks for removing that! lol

File: app/views/dealers/show.html.erb (hunk 1 above)
Comment (krislitman) [reply app/views/dealers/show.html.erb]: @abreaux26 we probably don't need them displayed actually, good call!

**Comment by abreaux26** [comment]
(Insert cool gif here)
It wouldn't let me paste 😭

**Pull request merged**